build.sbt

```diff
@@ -1,5 +1,17 @@
 scalaVersion := "2.12.10"
 
+val scalapbVersion = "0.10.2"
+
+lazy val codeGen = project
+  .in(file("code-gen"))
+  .settings(
+    name := "grpc-web-code-gen-with-metadata",
```

Comment:
I think grpcweb should have a single generator that can do with and without metadata through a generator parameter, so we don't rely on the default one only in one case.

Reply:
@thesamet The current generator only creates Service class and uses the default Req Res and Proto from default generator similar to zio-grpc, Do you mean we need a Single generator which does all of these?